code/game/machinery/doors/airlock_electronics.dm

```diff
@@ -17,7 +17,7 @@
 	var/autoset = TRUE // Whether the door should inherit access from surrounding areas
 
 /obj/item/airlock_electronics/attack_self(mob/user)
-	if(!ishuman(user) || !issilicon(user))
+	if (!ishuman(user) && !istype(user,/mob/living/silicon/robot))
```

Comment:
just use issillicon but with && instead?

Reply:
I guess. It looks like it's a different way of checking for if the user is a silicon. All I did was compare diffs between F19 and Bay12, this was the only difference. Put it in my build, tested it, it worked. No idea why.